Services/FlightService.cs

```diff
@@ -0,0 +1,141 @@
+﻿using Airport_Ticket_Booking_System.Models;
+using Airport_Ticket_Booking_System.Models.Enums;
+using Airport_Ticket_Booking_System.Repositories;
+using FluentValidation.Results;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace Airport_Ticket_Booking_System.Services;
+
+public static class FlightService
+{
+    public static void UploadFlights(string fileAddress)
+    {
+        if (FileSystemUtilities.IsFileValid(fileAddress))
+        {
+            try
+            {
+                FlightRepository.SaveFlightsFromFile(fileAddress);
+
+            }
+            catch (Exception e)
+            {
+
+                throw new Exception(e.Message);
+            }
```

Comment:
I think it's enough to do it this way.
```suggestion
            FlightRepository.SaveFlightsFromFile(fileAddress);
```